### PR TITLE
feature/11735_api_guardian

### PR DESCRIPTION
**Pull request by hasibskrijelj**
Update vendors

### CODE REVIEW

**Review by lvenier — APPROVED** [review]
$payer->setAuthorization('toto');
:-1:

**Comment by Zenzumai** [comment]
Sérieux? c'est quoi payer ?

**Review by CaillaudPA — APPROVED** [review]

File: src/Payer.php [hunk 1]
```diff
@@ -1,4 +1,5 @@
 <?php
+
 namespace Fei\Service\Payment\Client;
 
 use Fei\ApiClient\AbstractApiClient;
```
Comment (mtisserant) [inline src/Payer.php]: why not using AbstractApiClient from fei/api-client ?

**Review by mtisserant — CHANGES_REQUESTED** [review]
use if possible abstractApiClient from fei/api-client package

File: src/Payer.php [hunk 2]
```diff
@@ -59,8 +35,8 @@ public function request(Payment $payment)
             ->setUrl($this->buildUrl(self::API_PAYMENT_PATH_INFO));
 
         $request->setBodyParams(['payment' => \json_encode($payment->toArray())]);
-        if ($this->apiKey) {
-                $request->addHeader('Authorization', $this->getApiKey());
+        if ($this->getAuthorization()) {
+            $request->addHeader('Authorization', $this->getAuthorization());
```
Comment (mtisserant) [inline src/Payer.php]: this code is already write in send method of AbstractApiClient

**Review by mtisserant — APPROVED** [review]

**Pull request closed without merging**